Code/ElastixTransformixWrappers/src/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+set( ITK_NO_IO_FACTORY_REGISTER_MANAGER 1 )
+include( ${ITK_USE_FILE} )
+
+
+# TODO (Niels Dekker, LKEB, 2022) include(${ELASTIX_CONFIG_TARGETS_FILE})
+include(${CMAKE_BINARY_DIR}/../Elastix-build/ElastixTargets.cmake)
+
+add_library( ElastixImageFilter sitkElastixImageFilter.cxx sitkElastixImageFilterImpl.h sitkElastixImageFilterImpl.cxx )
+set_target_properties( ElastixImageFilter PROPERTIES SKIP_BUILD_RPATH TRUE )
+target_include_directories( ElastixImageFilter
+    PUBLIC
+        $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/Code/ElastixTransformixWrappers/include>
+        $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/Code/BasicFilters/include>
+        $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/Code/Common/include>
+        $<BUILD_INTERFACE:${CMAKE_BINARY_DIR}/Code/Common/include>
+        $<BUILD_INTERFACE:${CMAKE_BINARY_DIR}/../Elastix/Common/ParameterFileParser>
+        $<BUILD_INTERFACE:${CMAKE_BINARY_DIR}/../Elastix/Common/xout>
+        $<BUILD_INTERFACE:${CMAKE_BINARY_DIR}/../Elastix/Core/Configuration>
+        $<BUILD_INTERFACE:${CMAKE_BINARY_DIR}/../Elastix/Core/Install>
+        $<BUILD_INTERFACE:${CMAKE_BINARY_DIR}/../Elastix/Core/Kernel>
+        $<BUILD_INTERFACE:${CMAKE_BINARY_DIR}/../Elastix/Core/Main>
+        $<INSTALL_INTERFACE:${SimpleITK_INSTALL_INCLUDE_DIR}>
+    PRIVATE
+        ${CMAKE_CURRENT_SOURCE_DIR} )
+target_link_libraries( ElastixImageFilter PRIVATE elastix_lib )
```

Comment:
This dependency should be "INTERFACE".

If Elastix could be updated to use the modern CMake target properties all these include paths would not be needed.

Reply:
Again, addressed by this [force-push](https://github.com/SimpleITK/SimpleITK/compare/0920c1109b7fb55195248d145e5e953aae1012d6..574573628fac445ab0dcbcfe33724153594e3e7c), please check!